File: project/src/main/java/com/example/project/entities/Projects.java
package com.example.project.entities;

import java.io.Serializable;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.DBRef;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;


@Data
@Document(collection = "Projects")
public class Projects implements Serializable {
	@Id
	private int projects_id;
	private int budget;
	private Boolean internal;
	private String name;
	private Boolean still_going;
    private LocationProject locationProject;
	
}

File: project/angular_project/src/app/models/projects.ts
import { LocationProject } from "./location-project";

export class Projects {
  projects_id: number;
  budget: number;
  internal: boolean; 
  name: string;
  still_going: boolean; 
  locationProject: LocationProject;

  constructor() {
    this.projects_id = 0;
    this.budget = 0;
    this.internal = false;
    this.name = '';
    this.still_going = false;
    this.locationProject = new LocationProject();
  }
}